backend/dal/Configuration/ProjectStatusConfiguration.cs

```diff
@@ -23,6 +23,12 @@ public override void Configure(EntityTypeBuilder<ProjectStatus> builder)
 
             builder.Property(m => m.Description).HasMaxLength(1000);
 
+            builder.Property(m => m.Route).IsRequired();
+            builder.Property(m => m.Route).HasMaxLength(150);
+
+            builder.Property(m => m.Workflow).IsRequired();
```

Comment:
This property provides us a way to query for a subset of the project status.

Reply:
Could you please expand what this property is for? I'm not sure I follow what you mean here... thanks

Reply:
There are many project statuses, and most likely many more to come.  But we only want to extract the relevant status for a specific workflow (Submit Disposal Project).  This way when creating a new disposal project we can simply make a request for a specific workflow and we'll get back the steps/status that are relevant and will be displayed in the APP.

New Project -> Select Properties -> Update Information -> Required Documents -> ... -> Submit